chrome/content/zotero/zoteroPane.js

```diff
@@ -2945,6 +2981,14 @@ var ZoteroPane = new function()
 				case 'autoSync':
 					Zotero.Prefs.set(`sync.autoSync`, true);
 					break;
+				case 'quotaError':
+					if (options.library instanceof Zotero.Group) {
+						this.collectionsView.selectLibrary(options.library.libraryID);
```

Comment:
I don't think there's much point to focusing the group. We should probably provide the owner `userID` in the server response so that the client could know if it's the current user. If it was, it could take you to your account settings, and otherwise it should probably just use the space to show a longer message that says to contact the group owner.